packages/react/src/utils/useOutboundClickListener.tsx

```diff
@@ -22,13 +22,13 @@ const useOutboundClickListener = ({ trackLink }: MatomoInstance): void => {
           ),
         )
       ) {
-        // eslint-disable-next-line @typescript-eslint/no-unsafe-call
-        trackLink({ href })
+        // eslint-disable-next-line @typescript-eslint/no-unsafe-call,@typescript-eslint/no-unsafe-member-access
+        matomoInstance.trackLink.call(matomoInstance, { href })
```

Comment:
Why does the calling the method on the original object lose it's scope? Should the following code not work also?

```ts
matomoInstance.trackLink({ href })
```

Reply:
My guess is that the destructuring of the `useOutboundClickListener` function parameter makes the `trackLink ` function lose its context. I've changed the way the `trackLink` function is called. Also works.